Sources/Caching/KeyedDeferredValueStore.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  KeyedDeferredValueStore.swift
+//
+//  Created by Jacob Zivan Rakidzich on 8/12/25.
+
+import Foundation
+
+/// Some Task in which the value is Sendable and it can result in an error
+public typealias AnyTask<T: Sendable> = Task<T, Error>
```

Comment:
I don't think this should be `public`. If we need this in `RevenueCatUI`, declare it as `@_spi(Internal) public`

Reply:
Same for any other `public` that shouldn't be